src/ts/custom-mocha/bin/_mocha

```diff
@@ -1,6 +1,6 @@
-'use strict';
```

Comment:
I trust you on this one... but is there a reason why we have to maintain a mocha wrapper? High level, what are we doing that's "special"

Reply:
Yeah you kinda have to trust me here. Like trust me that the alternative would drive you up the wall.

The default mocha runner doesn't know anything about typescript - just javascript. So you need ts-node as a plugin, that compiles the ts on the fly. For starters, you just introduced a new way to compile ts in your project, which is itself and endless source of pain.

But then you realize that the compiled code uses ts-style path names, e.g. `app/framework`, so you need another plugin.

Oh but wait we need good error traces so we need appropriate sourcemaps in the transpiled js...and ts-node isn't quite doing what we want.

So once you get into having to understand the in's and out's of how these plugins are all configured, and what options they expose, or don't happen to expose, that support needs you define as critical, by diving into their source, etc, you get close to insanity.

This ts/node library world is a little nuts sometimes, and I've learned not to venture too far from "the metal" - defined as invocations of `tsc`, and then invoking the `node` binary.

If nothing else it's very reassuring to me knowing there's exactly *one way* to invoke `tsc` in this project.